component/authentication-endpoint/src/main/webapp/x509CertificateError.jsp

```diff
@@ -57,6 +57,21 @@
                     errorMessage = resourceBundle.getString("not.valid.certificate");
                 } else if (errorCode.equalsIgnoreCase("17003")) {
                     errorMessage = resourceBundle.getString("fail.validation.certificate");
+                } else if (errorCode.equalsIgnoreCase("17004")) {
```

Comment:
Is there any possibility to refer this through constants from a single location ?

Reply:
There was no imports from the authenticator to authenticator-endpoint. That might be to deploy the war file separately. hence It is risky to refer constants from a single location

Reply:
We can add the respective jar in web-inf/lib, when we deploy outside the Carbon.
Having said that, no problem to stick to current style.